src/web/layouts/StandardLayout.tsx

```diff
@@ -161,7 +160,7 @@ export const StandardLayout = ({ CAPI, config, NAV }: Props) => {
                 />
             </Section>
 
-            <CookieBanner />
+            <div data-island="cookie-banner" />
```

Comment:
I just went ahead and added a commit to change this to this simpler structure

Reply:
@oliverlloyd thanks for the review and correcting the use of a `<Section>` for this Island!